src/main/java/roomescape/common/exception/handler/GlobalExceptionHandler.java

```diff
@@ -1,35 +1,65 @@
 package roomescape.common.exception.handler;
 
+import io.jsonwebtoken.security.SignatureException;
 import jakarta.servlet.http.HttpServletRequest;
 import org.springframework.http.ResponseEntity;
 import org.springframework.http.converter.HttpMessageNotReadableException;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
+import roomescape.auth.exception.ForbiddenException;
+import roomescape.auth.exception.UnauthorizedException;
 import roomescape.common.exception.handler.dto.ExceptionResponse;
 
 @RestControllerAdvice
 public class GlobalExceptionHandler {
 
     @ExceptionHandler(NullPointerException.class)
-    public ResponseEntity<ExceptionResponse> handleNull(final NullPointerException exception, final HttpServletRequest request) {
+    public ResponseEntity<ExceptionResponse> handleNullPointer(final NullPointerException exception, final HttpServletRequest request) {
         ExceptionResponse exceptionResponse = new ExceptionResponse(
                 500, "[ERROR] " + exception.getMessage(), request.getRequestURI()
         );
 
         return ResponseEntity.internalServerError().body(exceptionResponse);
     }
 
-    @ExceptionHandler(value = IllegalArgumentException.class)
-    public ResponseEntity<ExceptionResponse> illegalArgument(final IllegalArgumentException exception, final HttpServletRequest request) {
+    @ExceptionHandler(IllegalArgumentException.class)
+    public ResponseEntity<ExceptionResponse> handleIllegalArgument(final IllegalArgumentException exception, final HttpServletRequest request) {
         ExceptionResponse exceptionResponse = new ExceptionResponse(
                 400, "[ERROR] " + exception.getMessage(), request.getRequestURI()
         );
 
         return ResponseEntity.badRequest().body(exceptionResponse);
     }
 
-    @ExceptionHandler(value = HttpMessageNotReadableException.class)
-    public ResponseEntity<ExceptionResponse> notReadable(
+    @ExceptionHandler(SignatureException.class)
+    public ResponseEntity<ExceptionResponse> handleSignature(final SignatureException exception, final HttpServletRequest request) {
+        ExceptionResponse exceptionResponse = new ExceptionResponse(
+                401, "[ERROR] " + exception.getMessage(), request.getRequestURI()
+        );
+
+        return ResponseEntity.status(401).body(exceptionResponse);
+    }
+
+    @ExceptionHandler(UnauthorizedException.class)
+    public ResponseEntity<ExceptionResponse> handleUnauthorize(final UnauthorizedException exception, final HttpServletRequest request) {
+        ExceptionResponse exceptionResponse = new ExceptionResponse(
+                401, "[ERROR] " + exception.getMessage(), request.getRequestURI()
+        );
+
+        return ResponseEntity.status(401).body(exceptionResponse);
```

Comment:
```suggestion
        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(exceptionResponse);
```
http status code도 매직넘버보다 상수를 활용하면 좋을 것 같아요~